tools/docker-go

```diff
@@ -75,16 +77,24 @@ for i in http_proxy https_proxy no_proxy HTTP_PROXY HTTPS_PROXY NO_PROXY ; do
   fi
 done
 
-docker run --rm \
-  -e GOCACHE='/tmp/.cache' \
-  -e GOPATH='/tmp/go' \
-  "${go_env[@]}" \
-  "${proxy_env[@]}" \
-  --user "$(id -u):$(id -g)" \
-  --security-opt label:disable \
-  ${DOCKER_RUN_ARGS} \
-  -v "${GO_MOD_CACHE}":/tmp/go/pkg/mod \
-  -v "${GO_MODULE_PATH}":/usr/src/host-ctr \
-  -w /usr/src/host-ctr \
-  "${SDK_IMAGE}" \
-    bash -c "${COMMAND}"
+IFS=','
+for dir in ${GO_MODULES_PATHS}; do
+  package=$(basename $dir)
+  cmd=$(echo ${COMMAND} | sed -e "s|MODULE|${package}|")
```

Comment:
I'd prefer duplication in `Makefile.toml` to this sort of magical substitution. This approach complicates reuse (c.f. #2053) since the command to run is no longer what's literally run.

One path forward might be to wrap the call to this script with a function in `Makefile.toml`.

Reply:
> magical substitution

I tend to agree. I didn't understand `MODULE` when I saw it earlier in Makefile.toml. I was asking "is that really the directory name?"